docker-compose.yml

```diff
@@ -0,0 +1,122 @@
+version: "1.0"
+
+services:
+  zookeeper:
+    image: bitnami/zookeeper:3.8.0
+    ports:
+      - 2181:2181
+    environment:
+      - ZOO_PORT_NUMBER=2181
+      - ZOO_TICK_TIME=2000
+      - ALLOW_ANONYMOUS_LOGIN=yes
+    networks:
+      - devenv
+    
+  kafka:
+    image: bitnami/kafka:3.1.0
+    ports:
+      - "9092:9092"
+    environment:
+      - KAFKA_BROKER_ID=0
+      - ALLOW_PLAINTEXT_LISTENER=yes
+      - "KAFKA_HEAP_OPTS=-Xmx500m -Xms500m"
+      - KAFKA_CFG_ZOOKEEPER_CONNECT=zookeeper:2181
+      - KAFKA_CFG_LISTENERS=EXTERNAL://:9092,CLIENT://:9093
+      - KAFKA_CFG_ADVERTISED_LISTENERS=EXTERNAL://localhost:9092,CLIENT://kafka:9093
+      - KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=CLIENT:PLAINTEXT,EXTERNAL:PLAINTEXT
+      - KAFKA_CFG_INTER_BROKER_LISTENER_NAME=CLIENT
+      - KAFKA_CFG_OFFSETS_TOPIC_REPLICATION_FACTOR=1
+      - KAFKA_CFG_DELETE_TOPIC_ENABLE=true
+      - KAFKA_CFG_MESSAGE_MAX_BYTES=3145729
+      - KAFKA_CFG_REPLICA_FETCH_MAX_BYTES=3145729
+    depends_on:
+      - zookeeper
+    networks:
+      - devenv
+
+  kafka-setup:
+    image: bitnami/kafka:2.8.1
+    hostname: kafka-setup
+    container_name: kafka-setup
+    depends_on:
+      - kafka
+    volumes:
+      - ./topics:/topics.txt
+      - ./create-topics.sh:/scripts/create-topics.sh
+    command: bash /scripts/create-topics.sh
+    environment:
+      KAFKA_CFG_BROKER_ID: ignored
+      KAFKA_CFG_ZOOKEEPER_CONNECT: ignored
+    networks:
+      - devenv
+  
+  get-kafka-exporter:
+    image: busybox:1.35.0
+    volumes:
+      - exporters:/exporters
+    command: wget --no-check-certificate https://repo1.maven.org/maven2/io/zeebe/zeebe-kafka-exporter/3.1.1/zeebe-kafka-exporter-3.1.1-jar-with-dependencies.jar -O /exporters/zeebe-kafka-exporter.jar
+
+  zeebe:
+    image: camunda/zeebe:8.2.1
```

Comment:
Zeebe is crashlooping for me when trying to run this today:
```
Caused by: io.camunda.zeebe.util.jar.ExternalJarLoadException: Cannot load JAR at [/usr/local/zeebe/exporters/zeebe-kafka-exporter.jar]: is not readable
```
I can try running later if it's a local error but you should probably check what's going on.